app/components/Browser/Browser.tsx

```diff
@@ -16,6 +16,7 @@ interface BrowserProps {
     tabs: object;
     windows: object;
     windowId: any;
+    history?: Array<any>;
```

Comment:
is this optional? It's coming from the store so should at least be an empty array or?